nextroute/solver_parallel.go

```diff
@@ -32,3 +33,9 @@ func NewParallelSolver(
 	connect.Connect(con, &newParallelSolver)
 	return newParallelSolver(model)
 }
+
+// Solve a nextroute vehicle routing problem.
+func Solve(ctx context.Context, solver ParallelSolver, options ParallelSolveOptions, solutions chan<- any) error {
```

Comment:
This is sugar code which removes some flexibility (changing the output formatter). I don't think we need this sugar and I would like to have the possibility of changing the formatter.